security/cmd/node_agent_k8s/main.go

```diff
@@ -38,15 +39,19 @@ var (
 		RunE: func(c *cobra.Command, args []string) error {
 			stop := make(chan struct{})
 
-			// TODO(quanlin): use real caClient when it's ready.
-			caClient := &mockCAClient{}
+			caClient, err := ca.NewCAClient(serverOptions.CAEndpoint, serverOptions.CARootFile)
+			if err != nil {
+				log.Printf("failed to create caClient: %v", err)
```

Comment:
I think we should use istio.io/istio/pkg/log

And log.Warnf()/Errorf() here, same below

Reply:
thanks,done